app/models/ems_refresh/parsers/openstack.rb

```diff
@@ -65,23 +65,23 @@ def ems_inv_to_hashes
     private
 
     def servers
-      @servers ||= @connection.servers_for_accessible_tenants
+      @servers ||= @connection.handled_list(:servers)
```

Comment:
Will it be ok to add the metamagic later? There are not that many calls right now to change it.

Not sure now whether to specify the list of the methods, that will be delegated through handler, or just do all. Basically all calls should be paginated, so it should not be that hard.

But it has to catch @connection.servers as well as @connection.servers.all(...)